modules/vaos/app/models/ccra/referral_detail.rb

```diff
@@ -14,28 +14,26 @@ class ReferralDetail
     # Initializes a new instance of ReferralDetail.
     #
     # @param attributes [Hash] A hash containing the referral details from the CCRA response.
-    # @option attributes [Hash] "Referral" The main referral data container.
     def initialize(attributes)
-      referral = attributes['Referral']
-      return if referral.blank?
+      return if attributes.blank?
 
-      @expiration_date = referral['ReferralExpirationDate']
-      @category_of_care = referral['CategoryOfCare']
-      @treating_facility = referral['TreatingFacility']
-      @referral_number = referral['ReferralNumber']
-      @referral_date = referral['ReferralDate']
-      @station_id = referral['StationID']
+      @expiration_date = attributes['referralExpirationDate']
```

Comment:
are we sure these keys* won't be symbolized? The `EpsAppointment` model is setup to access similar data via symbols: https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/vaos/app/models/vaos/v2/eps_appointment.rb